samigo/samigo-api/src/java/org/sakaiproject/tool/assessment/data/dao/grading/MediaData.java

```diff
@@ -55,6 +57,10 @@
 	private String lastModifiedBy;
 	private Date lastModifiedDate;
 	private String duration;
+	// Transient field to hold the ContentResource for media storage.
+	// When AssessmentGradingFacadeQueries.saveMedia is called, the byte array is written
+	// to a resource, rather than the blob column.
+	private ContentResource contentResource;
```

Comment:
I originally had the JPA annotation here, but we're not using any of the annotation-based mapping. The transient keyword is not really relevant either, because we are not concerned with serialization (and actually, it would be fine to serialize the ContentResource if sending over JMS or something).

We just don't want to persist this field, so I think the comment is appropriate here. If anything, the word transient could be substituted with nonpersisted or some such, but I think "transient" is still the best term; this just doesn't follow the exact Java/JPA-annotation mechanics.